util/make_timeline.py

```diff
@@ -134,9 +191,62 @@ def parse_file(args):
 
     return entries, last_ability_time
 
+def find_fights_in_file(file):
+    e_starts = []
+    encounter_start_staging = 0 # Staged to avoid spurious starts such as limit break usage
```

Comment:
Are `encounter_start_staging != 0` and `encounter_in_progress` redundant with each other? Just wondering if `encounter_in_progress` could get removed.

Reply:
That definitely could be clearer. I don't think they're functionally identical, since `in_progress` is explicitly for binary state, while `staging` contains formatted data. 

An example of where this could be important is use of the limit break on trash between bosses. This would send a 0000 limit break line, triggering a fight start. This fight start would either show the name of the dungeon, not the boss zone, or the previous boss zone. The party would then get to the next boss, which would trigger a new fight start, since zone seals have priority over everything else. However, the spurious fight start would already have been placed in the `encounter_starts` list, meaning that we would have an imbalance between the start and stop lists. Having it staged means that the spurious fight gets discarded without any extra work on our part.

To put it another way, `in_progress` is used specifically in relation to parsing unknown data, while `staging` has to do with storage and presentation of already-known data. It's likely that we *could* change things around some to negate the need for two separate variables here, but that would then require us to do more manipulation of the `starts` list, and I'm not sure whether that's worth it.

I'm totally fine with combining it if there's a simpler way to discard trash pulls It's very possible that once I get things worked out with `15/16` line starts vs `21` ending messages, we could combine them. But as it stands, we do need the two separate.

Reply:
Well, the specific start line (0000 limit break) is gone now, but the reason for this remains true even after use of `15/16` log lines, so I think it's worth maintaining the two different variables here.